upup/pkg/fi/vfs_castore.go

```diff
@@ -147,24 +145,24 @@ func (c *VFSCAStore) buildPrivateKeyPath(name string, id string) vfs.Path {
 	return c.basedir.Join("private", name, id+".key")
 }
 
-func (c *VFSCAStore) parseKeysetYaml(data []byte) (*kops.Keyset, KeysetFormat, error) {
+func (c *VFSCAStore) parseKeysetYaml(data []byte) (*kops.Keyset, bool, error) {
 	defaultReadVersion := v1alpha2.SchemeGroupVersion.WithKind("Keyset")
 
 	object, gvk, err := kopscodecs.Decode(data, &defaultReadVersion)
 	if err != nil {
-		return nil, "", fmt.Errorf("error parsing keyset: %v", err)
+		return nil, false, fmt.Errorf("error parsing keyset: %v", err)
 	}
 
 	keyset, ok := object.(*kops.Keyset)
 	if !ok {
-		return nil, "", fmt.Errorf("object was not a keyset, was a %T", object)
+		return nil, false, fmt.Errorf("object was not a keyset, was a %T", object)
 	}
 
 	if gvk == nil {
-		return nil, "", fmt.Errorf("object did not have GroupVersionKind: %q", keyset.Name)
+		return nil, false, fmt.Errorf("object did not have GroupVersionKind: %q", keyset.Name)
 	}
 
-	return keyset, KeysetFormat(gvk.Version), nil
+	return keyset, gvk.Version != KeysetFormatV1Alpha2, nil
```

Comment:
this seems to be a pretty important use of the constant that is buried in the middle of this file. Any thoughts on having something like
```go
const KeysetFormatLatest = KeysetFormatV1Alpha2
```

defined at the top of the file and referencing it here? I'm debating if that would improve readability or hinder it...

Reply:
In fact this might be the only use of `KeysetFormatV1Alpha2` now, so maybe it makes sense to get rid of it and just have this in ca.go:

```go
const (
  KeysetFormatV1Latest KeysetFormat = "v1alpha2"`
)

Reply:
I've been considering inlining the const since this is the only remaining reference. It certainly doesn't need to be public anymore.

Reply:
ok, i just want to make sure i understood what was happening here. Inlining the const could happen in a followup PR 👍